pkg/kritis/review/review.go

```diff
@@ -147,7 +146,7 @@ func (r Reviewer) findUnsatisfiedAuths(image string, auths []v1beta1.Attestation
 	notAttestedBy := []v1beta1.AttestationAuthority{}
 	for _, auth := range auths {
 		transport := AttestorValidatingTransport{Client: r.client, Attestor: auth}
-		attestations, err := transport.GetValidatedAttestations(image)
+		attestations, err := transport.GetValidatedAttestations(image, auths)
```

Comment:
This is inside a loop which goes over 'auths', so it's repeatedly getting attestations for all auths.  And there's in fact no need to change the signature of GetValidateAttestations at all, because transport already knows the 'auth'.

I'd further suggest using AttestationAuthority arguments in lower level client libraries (instead of array of AttestationAuthority).